geoPlugin integration and fix with empty Typeahead data

Dennisparchkov opened the pull request:
geoplugin and Enrich modification to work with current version of Rumal .
Fix bug with empty Typeahead Data.

Dennisparchkov committed:
Tyepahead data empty string when no data instead of None

Dennisparchkov committed:
geo plugin and enrich Fixes to work with current version of rumal

Dennisparchkov committed:
Fix report not working when typeahead data is empty, add null when empty.

pdelsante commented on interface/management/commands/enrich.py:
```diff
@@ -151,15 +147,17 @@ def handle(self, *args, **options):
                 try:
+                    data = self.get_data(task)
+                    self.mark_task_as_running(task)
+                    PluginTask(task=task, plugin_name='GeoPlugin').save()
```
Why are you explicitly calling the GeoPlugin name here? If I remember it correctly, the plugin should already be called by `make_ptask_queue` and `run_ptask_queue`.

I fear that explicitly calling the plugin may break things if the plugin is disabled in the conf.

Dennisparchkov committed:
Fix hardcoding the enrich daemon for GeoPlugin. Checks plugins available in plugin condif directory

Dennisparchkov commented on interface/management/commands/enrich.py:
```diff
@@ -98,6 +98,11 @@ def mark_ptask_as_completed(self, ptask):
         ptask.status = STATUS_COMPLETED
         ptask.save()
 
+    def save_ptasks(self, task):
```
This function adds the PluginTask for all available plugins

Dennisparchkov commented:
I added the general function that creates the PluginTask for all available plugins when a task is needed for enrichment. 
I also modified `plug.py`, it know returns the available plugins with a config file.
